flask_accept/test/app_for_testing.py

```diff
@@ -92,3 +97,44 @@ def get_v2(self):
 
 api.add_resource(IndexResourceWithoutFallback, '/resource/without-fallback')
 api.add_resource(IndexResourceWithFallback, '/resource/with-fallback')
+
+
+# Routes set up with flask_restplus
+class PlusResourceWithoutFallback(PlusResource):
+    @accept('application/vnd.vendor.v1+json')
+    def get(self):
+        """
+            The doc string of GET /plus/without-fallback
+        """
+        return jsonify(version='v1')
+
+    @get.support(
+        'application/json',
+        'application/vnd.vendor+json',
+        'application/vnd.vendor.v2+json')
+    def get_v2(self):
+        return jsonify(version='v2')
+
+
+class PlusResourceWithFallback(PlusResource):
+    @accept_fallback
+    def get(self):
+        """
+            The doc string of GET /plus/with-fallback
+        """
+        return jsonify(version='v0')
+
+    @get.support('application/vnd.vendor.v1+json')
+    def get_v1(self):
+        return jsonify(version='v1')
+
+    @get.support(
+        'application/json',
+        'application/vnd.vendor+json',
+        'application/vnd.vendor.v2+json')
+    def get_v2(self):
+        return jsonify(version='v2')
+
+
+plus_api.add_resource(PlusResourceWithoutFallback, '/plus/without-fallback')
+plus_api.add_resource(PlusResourceWithFallback, '/plus/with-fallback')
```

Comment:
Would you mind moving these tests to a separate file, something like `test_flask_restplus_integration.py`? (Bonus points if you do something similar with the `flask_restful` tests in this file as well.